cmd/zapi/cmd/get/get.go

```diff
@@ -0,0 +1,188 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/brimsec/zq/cmd/zapi/cmd"
+	"github.com/brimsec/zq/emitter"
+	"github.com/brimsec/zq/pkg/fs"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/zql"
+	"github.com/mccanne/charm"
+)
+
+var Get = &charm.Spec{
+	Name:  "get",
+	Usage: "get [options] <search>",
+	Short: "perform zql searches",
+	Long:  "TODO",
+	New:   New,
+}
+
+func init() {
+	cmd.CLI.Add(Get)
+	cmd.Get = Get
+}
+
+type Command struct {
+	*cmd.Command
+	zio.WriterFlags
+	// format     string
+	protocol   string
+	dir        string
+	outputFile string
+	reverse    bool
+	stats      bool
+	warnings   bool
+	wire       bool
+	final      *api.SearchStats
+}
+
+func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
```

Comment:
Will you please remove all of the ununsed options?

Reply:
I thought I got them all, let me take another look.

Reply:
Looks like they are all connected. The zio.WriterFlags obscure some of this.